common/lib/xmodule/xmodule/js/src/video/035_video_accessible_menu.js

```diff
@@ -80,7 +81,8 @@ function () {
 
         if (value) {
             state.videoAccessibleMenu.setValue(value);
-            button.text(gettext(msg));
+            // button.text(gettext(msg));
```

Comment:
Why is this commented out? If it is no longer needed, delete the code (if there is some value to having the commented out code, add a comment explaining why).